devito/dle/blocking_utils.py

```diff
@@ -13,15 +15,21 @@
 __all__ = ['BlockDimension', 'fold_blockable_tree', 'unfold_blocked_tree']
 
 
-def fold_blockable_tree(node, blockinner=True):
+def fold_blockable_tree(iet, blockinner=True):
```

Comment:
Note for reviewers: changes to this routine are a bug fix to avoid cross-loop blocking over boundary conditions loops